server/src/main/java/io/crate/types/TimestampType.java

```diff
@@ -68,7 +68,7 @@ public final class TimestampType extends DataType<Long>
         TimestampType::parseTimestampIgnoreTimeZone,
         Precedence.TIMESTAMP);
 
-    private static final StorageSupport<Long> STORAGE = new StorageSupport<>(true, true, new LongEqQuery()) {
+    private static final StorageSupport<Long> STORAGE = new StorageSupport<>(true, false, true, new LongEqQuery()) {
```

Comment:
Any reason not to support it for the `Timestamp` type?

Reply:
Not really, it was just not mentioned on the issue.
Enabled also for timestamp & timestamp_tz.